app/views/collections/show.html.erb

```diff
@@ -27,6 +27,11 @@ Unless required by applicable law or agreed to in writing, software distributed
   <div class="row">
     <div class="document-thumbnail col-sm-4 col-sm-push-8">
       <%= image_tag @doc_presenter.poster_url %>
+      <div class="filter-collection-button-wrapper">
+        <a href="<%= root_url + '?f[collection_ssim][]=' + @doc_presenter.name %>" class="btn btn-default btn-block">
```

Comment:
This is going to the root path which is now the new home page.  I think it should be changed to the following which goes to `/catalog`:
```suggestion
        <a href="<%= search_catalog_path('f[collection_ssim][]' => @doc_presenter.name) %>" class="btn btn-default btn-block">
```